src/analyze.ts

```diff
@@ -1196,6 +1196,14 @@ export default async function analyze(
       return;
     }
 
+    if (
+      'value' in staticChildValue &&
+      typeof staticChildValue.value === 'string' &&
+      staticChildValue.value.startsWith('.')
+    ) {
+      staticChildValue.value = path.resolve(dir, staticChildValue.value);
```

Comment:
Are we matching behavior here? https://github.com/vercel/next.js/pull/84155 seems like this is diverging

Reply:
Mmm - yeah, I have it wrong here, I should use `cwd` - and update the test

Reply:
In this branch, running `npm run build && node out/cli.js print test/unit/fs-emission/input.js` won't output `asset1.txt` cuz it doesn't exist in the root. 

If we place a dummy `asset1.txt` at the root and re-run, then we trace the file.

Reply:
I think we have two variations one with `cwd` at the root and once in the fixture